app/cdap/components/PostRunActions/PostRunActionsWizard.tsx

```diff
@@ -0,0 +1,301 @@
+                setShowValidateButton={this.setShowValidateButton}
+                propertyErrors={this.state.propertyErrors}
+                validatePluginProperties={this.props.validatePluginProperties}
+                validateCallback={this.validateCallback}
+                onValidateStart={this.onValidateStart}
+              />
+            ) : null}
+            {action && this.props.mode === 'view' ? (
+              <div className="postrun-configuration-content">{this.renderBody()}</div>
+            ) : null}
+            {this.props.mode === 'create' ? (
+              <WizardNav
+                mode={this.props.mode}
+                currentStage={currentStage}
+                artifact={this.props.artifact}
+                setAction={this.setActionHandler}
+                actionCreator={this.props.actionCreator}
+                toggle={this.toggleAndPreventPropagation}
+                getPostActions={this.props.getPostActions}
+                setShowValidateButton={this.setShowValidateButton}
+                propertyErrors={this.state.propertyErrors}
+                validatePluginProperties={this.props.validatePluginProperties}
+                validateCallback={this.validateCallback}
+                onValidateStart={this.onValidateStart}
+              />
+            ) : null}
+          </ModalBody>
+          <ModalFooter>
+            <div
+              className="btn btn-blue float-right close-button"
+              onClick={this.toggleAndPreventPropagation}
+            >
+              Close
+            </div>
+          </ModalFooter>
+        </Modal>
+      );
+    }
+
+    return (
+      <Modal
+        isOpen={this.props.isOpen}
+        toggle={this.toggleAndPreventPropagation}
+        backdrop="static"
+        modalClassName="post-run-actions-modal hydrator-modal node-config-modal"
+        zIndex={MATERIAL_UI_Z_INDEX + 1} // Workaround for material-ui PipelineModeless
+      >
+        {/* Not using <ModalHeader> here because it wraps the entire header in an h4 */}
+        <div className="modal-header">
+          <h4 className="modal-title float-left">
+            <span>{action.plugin.name || action.name}</span>
+            <small className="plugin-version">
+              {action.version || action.plugin.artifact.version}
+            </small>
+            <p>
+              <small>{action.description}</small>
+            </p>
+          </h4>
+          <div className="btn-group float-right">
+            <a className="btn" onClick={this.toggleAndPreventPropagation}>
+              <IconSVG name="icon-close" />
+            </a>
+          </div>
+        </div>
+        <ModalBody>
+          <div className="postrun-configuration-content">{this.renderBody()}</div>
+        </ModalBody>
+        <ModalFooter>
+          <div
+            className="btn btn-blue float-right close-button"
+            onClick={this.toggleAndPreventPropagation}
```

Comment:
I know this is copied over but please add https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role the accessibility attributes - divs should not be used as buttons and need extra attributes to be accessible